src/libs/filter-utils.ts

```diff
@@ -10,23 +10,25 @@ export namespace FilterUtilities {
    * @returns If specified dates were null or not, if specified date parameters were between Forecast dates or if currentDate is between Forecast dates
    */
   export const filterByDate = (dateRange: DateRange, currentDate: Date, parameters: { startDate?: Date, endDate?: Date }): boolean => {
-    if (dateRange.start_date === null || dateRange.end_date === null) {
+    if (dateRange.start_date === null) {
       return false;
     }
-
+    const start_date = new Date(dateRange.start_date);
+    let end_date = new Date(dateRange.end_date);
```

Comment:
I'm not sure why DateRange was set up with keys using snake_case but lets at least use camelCase for these two variables here

Reply:
I would add a line break after these variables for code clarity